tests/native_tests/language.hy

```diff
@@ -340,19 +340,29 @@
 
 (defn test-cond []
   (cond
-   [(= 1 2) (assert (is True False))]
-   [(is None None) (setv x True) (assert x)])
+    (= 1 2) (assert (is True False))
+    (is None None) (do (setv x True) (assert x)))
   (assert (= (cond) None))
 
   (assert (= (cond
-    [False]
-    [[]]
-    [8])) 8)
+    False 1
+    [] 2
+    True 8)) 8)
```

Comment:
so it turns out this paren is one short and we need to slurp that 8 up to actually check equality

Reply:
Good catch. Fixed.